pkg/cluster/pod.go

```diff
@@ -155,7 +155,7 @@ func (c *Cluster) masterCandidate(oldNodeName string) (*v1.Pod, error) {
 	}
 
 	if len(replicas) == 0 {
-		c.logger.Warningf("single master pod for cluster %q, migration will cause disruption of the service")
+		c.logger.Warningf("single master pod for cluster %q, migration will cause disruption of the service", c.Name)
```

Comment:
is this needed? dont we have the log context displaying this?

Reply:
I've already fixed that as a part of https://github.com/zalando-incubator/postgres-operator/pull/281/ (and the working here is confusing, since it could be interpreted as something related to the Kubernetes service).